### PR TITLE
Add document for node selector feature

### PR DESCRIPTION
**Pull request by PhanLe1010**
longhorn/longhorn#2199

### CODE REVIEW

**Comment by PhanLe1010** [comment]
Just in case you haven't notice, we can preview this PR on a live website by clicking on the highlighted linked as shown in the picture:
Screen Shot 2021-04-14 at 10 51 15 PM

File: content/docs/1.1.1/advanced-resources/deploy/node-selector.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+   > Note: Because of the limitation of Rancher 2.5.x, if you are using Rancher UI to install Longhorn, you need to click `Edit As Yaml` and add setting `systemManagedComponentsNodeSelector` to `defaultSettings`.
+   >
+   > For example:
+   > ```yaml
+   > defaultSettings:
+   >   systemManagedComponentsNodeSelector: "label-key1:label-value1"
+   >  ```
+
+
+### Setting up Taints and Tolerations After Longhorn has been installed
```
Comment (innobead) [inline content/docs/1.1.1/advanced-resources/deploy/node-selector.md:47]:
```suggestion
### Setting up Node Selector After Longhorn has been installed
```

File: content/docs/1.1.1/advanced-resources/deploy/node-selector.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+> * Since all Longhorn components will be restarted, the Longhorn system is unavailable temporarily.
+> * Make sure all Longhorn volumes are `detached`. If there are running Longhorn volumes in the system, this means the Longhorn system cannot restart its components and the request will be rejected.
+> * Don't operate the Longhorn system while node selector settings are updated and Longhorn components are being restarted.
+
+1. Prepare
+   * If you are changing node selector in a way so that Longhorn cannot run on some nodes that Longhorn is currently running on,
+     you will lose the volume replicas on those nodes.
+     Therefore, It is recommended that you evict replicas and disable scheduling for those nodes before changing node selector.
+     See [Evicting Replicas on Disabled Disks or Nodes](../../../volumes-and-nodes/disks-or-nodes-eviction) for more details about how to do this.
+   * Stop all workload and detach all Longhorn volumes. Make sure all Longhorn volumes are `detached`
```
Comment (innobead) [inline content/docs/1.1.1/advanced-resources/deploy/node-selector.md:59]:
```suggestion
   * Stop all workloads and detach all Longhorn volumes. Make sure all Longhorn volumes are `detached`.
```

**Review by innobead — CHANGES_REQUESTED** [review]
LGTM, just some comments needed to resolve.

**Comment by innobead** [comment]
cc @catherineluse please help review and approve. thanks.

File: content/docs/1.1.1/advanced-resources/deploy/node-selector.md (hunk 1 above)
Comment (PhanLe1010) [reply content/docs/1.1.1/advanced-resources/deploy/node-selector.md]: Thank you. Fixed

File: content/docs/1.1.1/advanced-resources/deploy/node-selector.md (hunk 2 above)
Comment (PhanLe1010) [reply content/docs/1.1.1/advanced-resources/deploy/node-selector.md]: Thank you. Fixed

**Review by innobead — APPROVED** [review]

**Review by shuo-wu — APPROVED** [review]

**Comment by jenting** [comment]
> Just in case you haven't noticed, we can preview this PR on a live website by clicking on the highlighted linked as shown in the picture:
> Screen Shot 2021-04-14 at 10 51 15 PM

cool!!!

**Pull request merged**